src/sites/twitch-twilight/modules/host-options.vue

```diff
@@ -62,12 +56,12 @@
 										:checked="autoHostSettings.enabled"
 										@change="updateCheckbox">
 									<label for="autoHostSettings:enabled" class="tw-checkbox__label">
-										Auto Hosting
+										{{ t('metadata.host.setting.auto-hosting.title', 'Auto Hosting') }}
 									</label>
 								</div>
-								<section class="c-text-alt-2" style="padding-left: 2.2rem;">
-									Toggle all forms of auto hosting: teammates, host list, and similar channels.<br>
-									<a href="https://blog.twitch.tv/grow-your-community-with-auto-hosting-e80c1460f6e1" target="_blank" rel="noopener">Learn More</a>
+								<section class="c-text-alt-2 checkbox-description">
```

Comment:
I like to prefix classes with `ffz-` to avoid potential collisions with Twitch/other extensions if they do stuff to their CSS.